### PR TITLE
DOCSP-31499 Release notes, source constant updates for v1.9.2

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [JIRA](https://jira.mongodb.org/browse/DOCSP-31499)
- [Stage](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-31499/atlas-cli-changelog/)
- [Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64ba884487b3b36046b6aa3c) (has errors)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -14,6 +14,19 @@
...
    :class: singlecol
 
+.. _atlas-cli-1.9.2:
+
+{+atlas-cli+} 1.9.2
+-------------------
+
+Released 20 July 2023
+
+- Fixes issue with printing table format for the
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:24]:
```suggestion
- Fixes issue with the printed table format for the
```

**Review by sarahsimpers — APPROVED** [review]
Small copy nit then LGTM

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -14,6 +14,19 @@
...
+
+{+atlas-cli+} 1.9.2
+-------------------
+
+Released 20 July 2023
+
+- Fixes issue with the printed table format for the
+  :ref:`atlas-networking-containers-list` command. 
+- Removes support for MongoDB 4.2.
+- Configures :ref:`atlas-quickstart` command as an alias for the
```
Comment (blva) [inline source/atlas-cli-changelog.txt:27]:
```suggestion
- Configures :ref:`atlas-quickstart` command as an alias for the
- Fixes issue with backup restore watch commands
```

Comment (blva) [reply source/atlas-cli-changelog.txt:27]: sorry, adding a suggestion here of one release not that we missed.

**Review by blva — CHANGES_REQUESTED** [review]

**Review by blva — APPROVED** [review]
LGTM thanks for the changes!

**Pull request merged**